docs/analyze/visualize-results.ipynb

```diff
@@ -0,0 +1,166 @@
+      "text/plain": [
+       "<Figure size 700x500 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "plot_histogram(quasi_dists)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57a1f479-3a5c-4eb4-a550-48c12d8f5c17",
+   "metadata": {},
+   "source": [
+    "### Options when plotting a histogram\n",
+    "\n",
+    "Use the following options for `plot_histogram` to adjust the output graph.\n",
+    "\n",
+    "* `legend`: Provides a label for the executions. It takes a list of strings used to label each execution's results. This is mostly useful when plotting multiple execution results in the same histogram\n",
+    "* `sort`: Adjusts the order the bars in the histogram are rendered. It can be set to either ascending order with `asc` or descending order with `desc`\n",
```

Comment:
```suggestion
    "* `sort`: Adjusts the order of the bars in the histogram. It can be set to either ascending order with `asc` or descending order with `desc`\n",
```